packages/astro/src/cli/add/index.ts

```diff
@@ -813,11 +808,11 @@ async function validateIntegrations(
 				};
 			}),
 		);
-		spinner.success();
+		spinner.stop();
```

Comment:
Quick comparison of the two states here:

| Current CLI | This PR |
|--------|--------|
| <img width="179" height="23" alt="Checkmark icon Resolving packages..." src="https://github.com/user-attachments/assets/fb79bc61-06f7-46ea-9d1e-d5d8cbe21426" /> | <img width="253" height="29" alt="diamond icon with no text" src="https://github.com/user-attachments/assets/6be7e406-6ab5-4adc-a7fb-b7d3815562c1" /> | 

The “Resolving packages” message is preserved currently as a reference to the completed task, whereas it looks like Clack’s `stop()` clears that out. I think it would be nice to preserve a message to refer back to.

Maybe as simple as?
```suggestion
		spinner.stop('Resolved packages');
```